python/hpsmc/generators.py

```diff
@@ -284,6 +290,12 @@ def make_param_card(self, param_card):
                 data[i] = "      624 %.7fe-03 # mpid" % (self.mpid) + '\n'
             if "mrhod" in data[i] and self.mrhod is not None:
                 data[i] = "      625 %.7fe-03 # mrhod" % (self.mrhod) + '\n'
+            if "Map" in data[i] and "ap :" not in data[i] and self.Map is not None:
+                data[i] = "    1 %.7e # Map\n" % (self.Map/1000)
+            if "Mchi" in data[i] and "dMchi" not in data[i] and self.mchi is not None:
+                data[i] = "    1 %.7e # Mchi\n" % (self.mchi/1000)
+            if "dMchi" in data[i] and "Mchi/2" not in data[i] and self.dmchi is not None:
+                data[i] = "    2 %.7e # dMchi" % (self.dmchi/1000)
```

Comment:
Can you confirm this doesn't break SIMPs or A'?

Reply:
Yes, neither `generators/madgraph4/src/ap/param_card.dat` nor `generators/madgraph5/src/simp/Cards/param_card.dat` have a `Mchi` in them.
```
$ grep Mchi generators/madgraph4/src/ap/param_card.dat generators/madgraph5/src/simp/Cards/param_card.dat
$
```

I don't think it is worthwhile to rerun the full pipeline because SDF is moving really slowly and we can plainly see that the derived param_card.dat files will not be affected by this change.

Reply:
I don't think we need to run the full pipeline, just generate an lhe file for simps and for A' and confirm it doesn't crash

Reply:
should be a quick check that can be ran locally.